example/lib/onboarding_flow/onboarding_flow.dart

```diff
@@ -48,9 +48,7 @@ class Step1 extends StatelessWidget {
         children: [
           FloatingActionButton(
             heroTag: 0,
-            onPressed: () {
-              context.flow<OnboardingSteps>().complete((_) => null);
-            },
+            onPressed: context.flow<OnboardingSteps>().complete,
```

Comment:
```suggestion
            onPressed: () => context.flow<OnboardingSteps>().complete(),
```

I would prefer if we kept this as a closure because otherwise on each build we are performing a lookup unnecessarily.